LiskHQ/lisk-hub
# Remove unused redux-logger dependency - Closes #721

### What was the problem?
- some dependencies listed in package and not used
### How did I fix it?
- find out which ones, (see issue number)
- removed some of the unused ones
### How to test it?
```
rm -rf ./node_modules
npm i
npm run build
npm run dev
```
should work as expected.

### Review checklist
- The PR solves #INSERT_ISSUE_NUMBER
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
